tests/coalaTest.py

```diff
@@ -179,3 +179,23 @@ def test_logged_error_causes_non_zero_exitcode(self):
             )
 
             assert exitcode == 1
+
+    def test_coala_no_color(self):
+        configure_logging(color=False)
+        with bear_test_module(), \
+                prepare_file(['#fixme'], None) as (lines, filename):
+            retval, stdout, stderr = execute_coala(
+                             coala.main,
+                             'coala', '-c', os.devnull,
+                             '-f', re.escape(filename),
+                             '-b', 'LineCountTestBear')
+            self.assertIn('This file has 1 lines.',
+                          stdout,
+                          'The output should report count as 1 lines')
+            self.assertIn('An unknown error occurred. This is a bug. '
```

Comment:
Why is this allowing stderr with `This is a bug. ... This should never happen.`
Why is there an unknown error?  Why is that a good thing?
If it really is a good thing, it definitely needs a comment to explain why this should be in the output.